packages/shared/src/components/auth/AuthSignBack.tsx

```diff
@@ -73,7 +73,7 @@ export const AuthSignBack = ({
         <ConditionalWrapper
           condition={provider !== 'password'}
           wrapper={(component) => (
-            <div className="relative">
+            <div className="relative" aria-hidden>
```

Comment:
Onboarding's profile image is not a required information on screen reader, maybe is better to skip just to the login button since image cannot be seen in those cases.